src/main/client/player/player-component.ts

```diff
@@ -0,0 +1,24 @@
+import { Inject, Singleton } from 'typescript-ioc';
+import { PlayerStore } from '../../shared/player/player-store';
+import { Player } from '../../shared/player/player-model';
+import { Subject } from 'rxjs';
+
+@Singleton
```

Comment:
can this handle multiple players?

Reply:
At this point I'm not sure if this component will handle the players as well as the client (player), so I started with the client and we will see.